### ISSUE TITLE
Incompatible with mapbox-gl (v3)

### ISSUE THREAD

**Issue by ronaldtveen** [studiometa/vue-mapbox-gl#134]
Hi,

Mapbox GL JS is currently @3.1.2 with @3.2 in beta.
When I try to install Vue Mapbox GL v2.3.4 it is incompatible with that version:

```sh
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: undefined@0.1.0
npm ERR! Found: mapbox-gl@3.1.2
npm ERR! node_modules/mapbox-gl
npm ERR!   mapbox-gl@"^3.1.2" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer mapbox-gl@"^2.8.2" from @studiometa/vue-mapbox-gl@2.3.4
npm ERR! node_modules/@studiometa/vue-mapbox-gl
npm ERR!   @studiometa/vue-mapbox-gl@"^2.3.4" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

I'd really like to update to that major 🚀
Is this in the works and will it be addressed soon?

**Comment by olegmorajko** [comment]
We have the same issue. Are you planning to support Mapbox GL v3.x ?

**Comment by titouanmathis** [comment]
This should be fixed with the release of [v2.4.0](https://github.com/studiometa/vue-mapbox-gl/releases/tag/2.4.0). I have not fully tested the compatibility, but the demo and docs are working without error. 

You should now be able to install this package along v3 of `mapbox-gl`.
